15979 Tabbing is missing for WP details view in IE 10

hschink opened the pull request:
[`* `#15979` Tabbing is missing for WP details view in IE 10`](https://community.openproject.org/work_packages/15979)

Commit:
Add empty hrefs

Empty hrefs allow IE10 to properly focus the elements.

TeatroIO commented:
I've prepared a stage. <a href='http://fix-15979-tabbing-is-missing-for-wp-details-view-tabs-in-ie-10.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

mfrister commented:
Looks good, but as for #2031, I'd like a comment on the empty hrefs.

Commit:
Add comment to hrefs with empty URLs

This should prevent fellow developers from accidentally removing the previous fix for IE 10.

hschink commented:
@meeee Thanks for the hint! Done with https://github.com/opf/openproject/commit/b7743cecd0381b2505723a762ee0b51c5c8ddc5c.

mfrister commented:
:+1:
